Cognite.Simulator.Extensions/SequencesExtensions.cs

```diff
@@ -157,27 +157,28 @@ public static class SequencesExtensions
         /// <param name="sequenceExternalId">Simulator integration sequence external id</param>
         /// <param name="update">Data to be updated, if init is set to true</param>
         /// <param name="token">Cancellation token</param>
+        /// <param name="lastLicenseCheckTimestamp"></param>
+        /// <param name="lastLicenseCheckResult"></param>
         /// <exception cref="SimulatorIntegrationSequenceException">Thrown when one or more sequences
         /// rows could not be updated. The exception contains the list of errors</exception>
-        public static async Task UpdateSimulatorIntegrationsHeartbeat(
+        public static async Task UpdateSimulatorIntegrationsData(
             this SequencesResource sequences,
             string sequenceExternalId,
             bool init,
             SimulatorIntegrationUpdate update,
-            CancellationToken token)
+            CancellationToken token,
+            long lastLicenseCheckTimestamp,
+            string lastLicenseCheckResult)
         {
             var rowsToCreate = new List<SequenceDataCreate>();
-            var rowData = new Dictionary<string, string>
-                {
-                    { SimulatorIntegrationSequenceRows.Heartbeat, $"{DateTime.UtcNow.ToUnixTimeMilliseconds()}" }
-                };
+            var rowData = new Dictionary<string, string>();
+
+            rowData.Add(SimulatorIntegrationSequenceRows.Heartbeat, $"{DateTime.UtcNow.ToUnixTimeMilliseconds()}");
+            rowData.Add(SimulatorIntegrationSequenceRows.LicenseTimestamp, $"{lastLicenseCheckTimestamp}");
+            rowData.Add(SimulatorIntegrationSequenceRows.LicenseStatus, lastLicenseCheckResult);
+
             if (init)
             {
-                if (update == null)
-                {
-                    throw new ArgumentNullException(nameof(update));
```

Comment:
@polomani It is impossible for init to be false and update to be null because of how they are made in a previous function.
`var update = init ?
                    new SimulatorIntegrationUpdate
                    {
                        Simulator = simulator.Name,
                        DataSetId = simulator.DataSetId,
                        ConnectorName = GetConnectorName(),
                        ConnectorVersion = GetConnectorVersion(),
                        SimulatorVersion = GetSimulatorVersion(simulator.Name),
                        ExtraInformation = GetExtraInformation(simulator.Name),
                        SimulatorApiEnabled = ApiEnabled()
                    }
                    : null;
                    await sequences.UpdateSimulatorIntegrationsHeartbeat(
                        _simulatorSequenceIds[simulator.Name],
                        init,
                        update,
                        token).ConfigureAwait(false);
                }`

Reply:
@kbrattli Yes, but we get a warning, it's better to get rid of it
![Screenshot 2023-07-24 at 14 07 49](https://github.com/cognitedata/dotnet-simulator-utils/assets/10908415/670f6e65-d989-4fff-9a8f-0f52c1f61446)